### ISSUE TITLE
Unknown formats being ignored in request validation

### ISSUE THREAD

**Issue by ViniGodoy** [cdimascio/express-openapi-validator#332]
**Describe the bug**
First of all, thanks for the amazing package! I've refactored my entire server around it! It's truly handy!
I'm using openAPI v3.0. 

The OpenID validator was configured as:
```javascript
const swaggerJsDoc = require('swagger-jsdoc');

const swaggerOptions = {
    swaggerDefinition: {
        openapi: "3.0.3",
        info: {
            title: "Cluster API",
            version: "1.0.0-alpha.7",
            description: "Cluster API documentation"
        },
        servers: [
            {
                url: process.env.SERVER_HOME + "/api",
                description: "Cluster server",
            }
        ]
    },
    apis: [
        "./code/component/**/*.yaml",
        "./code/component/**/*.js"
    ]
}

const swaggerDocs = swaggerJsDoc(swaggerOptions);

new OpenApiValidator({
    apiSpec: swaggerDocs,
    unknownFormats: ['hypertext'],
    operationHandlers: __dirname + "/../component"
}).install(app).then(() => {
    app.get('*', (req, res) => {
        res.sendFile('index.html', {root: path.join(__dirname, '../../ui/')});
    });
    app.use(error.handlers);
});
```

When validating this Schema:
 ```yaml
 components:
    schemas:
      FeaturedData:
        type: object
        properties:
          title:
            type: string
           example: "7 Ways to Make a Difference"
          content:
            type: string
            format: hypertext
            example: "<p>There are dozens of reason why..."
          date:
            type: string
            format: date-time
            example: "2017-09-08T03:00:00.000Z"
          topText:
            type: string
            example: "Thrive Global Presents"
          banner:
            $ref: '#/components/schemas/File'
          bottomText:
            type: string
            example: "Helping your community, helping yourself"
          order:
            type: integer
            format: int32
            example: "30"

```

Configured in this post path:

```yaml
 /featured:
   post:
      tags:
          - "featured"
      summary: "Inserts a new Featured content"
      description: "Inserts a new featured content. All object fields are mandatory."
      operationId: insert
      x-eov-operation-handler: featured/handler-featured
      requestBody:
        description: "Featured content to add"
        required: true
        content:
          "application/json":
            schema:
              $ref: "#/components/schemas/FeaturedData"
      responses:
          '201':
              description: "Sucessfull operation"
      security:
          - JWT: []
```

But, despite the unknownFormats being passed in the OpenAPIValidator constructor, I still get the following error:

```
Error: unknown format "hypertext" is used in schema at path "#/properties/content"
    at Object.generate_format [as code] (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/dotjs/format.js:69:15)
    at Object.generate_validate [as validate] (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/dotjs/validate.js:382:35)
    at Object.generate_properties [as code] (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/dotjs/properties.js:195:26)
    at generate_validate (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/dotjs/validate.js:382:35)
    at localCompile (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/compile/index.js:88:22)
    at Ajv.compile (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/compile/index.js:55:13)
    at Ajv._compile (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/ajv.js:348:27)
    at Ajv.getSchema (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/ajv/lib/ajv.js:204:54)
    at shouldHandle (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/express-openapi-validator/dist/middlewares/openapi.multipart.js:78:119)
    at /Users/vinigodoy/Documents/git/clustersrv/api/node_modules/express-openapi-validator/dist/middlewares/openapi.multipart.js:13:13
```

This even occurs when I switch the unknownFormats to the 'ignore' option.

**Comment by cdimascio** [comment]
@ViniGodoy thanks for the issue. working on a fix

**Comment by cdimascio** [comment]
fixed in `v3.16.5`

**Comment by ViniGodoy** [comment]
Wow, that was fast! 
Worked like a charm! Thanks!
